app/move/controllers/create/court-hearings.js

```diff
@@ -43,6 +44,37 @@ class CourtHearingsController extends CreateBaseController {
     }
   }
 
+  process(req, res, next) {
+    const { court_hearing__start_time: startTime } = req.form.values
+    const moveDate = req.sessionModel.get('date')
+    const hearingDate = parseISO(`${moveDate}T${startTime}`)
```

Comment:
This will give an invalid date if `startTime` is undefined. We can guarantee that `startTIme` will exist and be an empty string at least?

Reply:
Good point. I've moved that inside the `if` block to only need to do these if `startTime` exists.

If the date is not valid, it will fallback to the original value - for example, during failed validation